core/src/main/scala/caliban/Configurator.scala

```diff
@@ -0,0 +1,47 @@
+package caliban
+
+import caliban.CalibanError.ValidationError
+import caliban.execution.QueryExecution
+import caliban.validation.Context
+import caliban.validation.Validator.DefaultValidations
+import zio._
+import zio.prelude.EReader
+
+object Configurator {
+  case class ExecutionConfiguration(
+    skipValidation: Boolean = false,
```

Comment:
I feel like this might cause some confusion on how `skipValidation = true` differs from `validations = Nil`. I doubt we could get rid of `skipValidation` altogether, so I'm not sure what's the best way forward. Just a thought (probably not the best) - we could change `validations` into a `NonEmptyList` from prelude to perhaps prevent some confusions?

Reply:
It might be even more confusing if you provide `skipValidation = true` but you see you have to provide a non-empty list too. I will add a scaladoc with an explicit mention that `validations` are ignored when `skipValidation` is true.

Reply:
Sounds good!